### DISCUSSION TITLE
Group-hover visibility changes don't work when children contain SVGs

### DISCUSSION THREAD

**Discussion by pratikkabra-alation** [tailwindlabs/tailwindcss#10255] [Help] [answered]
I've run into an issue on Tailwind (3.2.4) with group-hover. I've been trying to get an SVG of a caret to appear (visible) when hovering over any part of the parent. This doesn't seem to work, and the issue specifically seems to be the use of the SVG.

Example here: https://play.tailwindcss.com/LgiQDbYH8K

When I swap the SVG out for just text (eg. "Hello"), the group-hover seems to work fine. But fails with the SVG.

I've tested in Chrome and Firefox, with the same result. Any thoughts?

**Comment by wongjn** [comment, marked as answer]
The `group-hover` *is* working fine, inspect with devtools to see that `visibility: visible` is being applied on hover. The actual issue you may be facing is the fact that the other SVGs apart from the first do no render anything visible. This is because of the repeated `<clipPath>` elements with the same `id`. The other SVGs will appear if all the `id` attribute values are unique. One could also forego the need for the `clip-path`-`<clipPath>` since all it is doing is clipping the SVG content to its view box, but the content is within the view box anyway so it is kind of obsolete. [Here is the updated example](https://play.tailwindcss.com/4l5E1xMelq) with the `id` attributes now unique and also an extra item demonstrating the removal of the `clip-path`.

**Reply by pratikkabra-alation** [reply]
Thanks! Didn't even occur to me that the colliding IDs would manifest the error in that way.
